### PR TITLE
Home task is 1 

### PR DESCRIPTION
**Pull request by Yaroslav-new**
Hi there,

I know, I've just done the first test, but I will do the second one soon ;)

you can take a look at the first one.

### CODE REVIEW

File: tests/my-first-test.spec.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test';
+import { exec } from 'child_process';
```
Comment (hlazkov) [inline tests/my-first-test.spec.js:2]: Lets remove this since we dont need it

File: tests/my-first-test.spec.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const standardUser = "standard_user";
+const password = "secret_sauce";
+const url = 'https://www.saucedemo.com/';
```
Comment (hlazkov) [inline tests/my-first-test.spec.js:6]: Lets move this to base url in playwright config, so you could set relative path into page.goto

File: tests/my-first-test.spec.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  test('Perform Login', async ({ page }) => {
+    await expect (page.locator('.title')).toBeVisible();
+    await expect (page.locator('.shopping_cart_link')).toBeVisible();
+    const productsItems = await (page.locator('.inventory_item')).count()
+    expect(productsItems).toBeGreaterThan(1);
+  });
+
+  test('Add product to the cart', async ({ page }) => {
+
```
Comment (hlazkov) [inline tests/my-first-test.spec.js:24]: I dont see anything here. please add a test.

File: tests/my-first-test.spec.js (hunk 1 above)
Comment (Yaroslav-new) [reply tests/my-first-test.spec.js]: Fixed

File: tests/my-first-test.spec.js (hunk 2 above)
Comment (Yaroslav-new) [reply tests/my-first-test.spec.js]: Fixed

File: tests/my-first-test.spec.js (hunk 3 above)
Comment (Yaroslav-new) [reply tests/my-first-test.spec.js]: Fixed

**Review by hlazkov — APPROVED** [review]
Well done here!

**Pull request merged**